pkg/kubelet/kubelet.go

```diff
@@ -654,6 +681,29 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 
 	klet.statusManager = status.NewManager(klet.kubeClient, klet.podManager, klet)
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletServerCertificate) {
+		ips, err := allLocalIPsWithoutLoopback()
+		if err != nil {
+			return nil, err
+		}
+		ips = append(ips, cloudIPs...)
```

Comment:
Is it possible for `allLocalIPsWithoutLoopback` to include the external IPs? If so, I think you should probably de-dupe this list?